src/test/kotlin/no/nav/familie/tilbake/dokumentbestilling/manuell/brevmottaker/ManuellBrevmottakerServiceTest.kt

```diff
@@ -0,0 +1,141 @@
+package no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker
+
+import io.kotest.assertions.throwables.shouldNotThrow
+import io.kotest.matchers.collections.shouldBeEmpty
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import io.mockk.slot
+import io.mockk.verify
+import no.nav.familie.kontrakter.felles.historikkinnslag.Aktør
+import no.nav.familie.tilbake.OppslagSpringRunnerTest
+import no.nav.familie.tilbake.api.dto.ManuellBrevmottakerDto
+import no.nav.familie.tilbake.behandling.BehandlingRepository
+import no.nav.familie.tilbake.behandling.FagsakRepository
+import no.nav.familie.tilbake.behandling.domain.Behandling
+import no.nav.familie.tilbake.data.Testdata
+import no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker.domene.MottakerType.DØDSBO
+import no.nav.familie.tilbake.historikkinnslag.HistorikkService
+import no.nav.familie.tilbake.historikkinnslag.TilbakekrevingHistorikkinnslagstype
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import java.time.LocalDateTime
+
+class ManuellBrevmottakerServiceTest : OppslagSpringRunnerTest() {
+
+    @Autowired
+    private lateinit var manuellBrevmottakerRepository: ManuellBrevmottakerRepository
+    private val mockHistorikkService: HistorikkService = mockk(relaxed = true)
+
+    @Autowired
+    private lateinit var fagsakRepository: FagsakRepository
+
+    @Autowired
+    private lateinit var behandlingRepository: BehandlingRepository
+
+    private lateinit var behandling: Behandling
+    private lateinit var manuellBrevmottakerService: ManuellBrevmottakerService
+    private val opprettetTidspunktSlot = slot<LocalDateTime>()
+
+    @BeforeEach
+    fun init() {
+        fagsakRepository.insert(Testdata.fagsak)
+        behandling = behandlingRepository.insert(Testdata.behandling)
+
+        manuellBrevmottakerService = ManuellBrevmottakerService(manuellBrevmottakerRepository, mockHistorikkService)
+
+        every {
+            mockHistorikkService.lagHistorikkinnslag(
+                behandlingId = any(),
+                historikkinnslagstype = any(),
+                aktør = any(),
+                opprettetTidspunkt = capture(opprettetTidspunktSlot)
+            )
+        } just runs
+    }
+
+    @Test
+    fun `leggTilBrevmottaker skal legge til DØDSBO brevmottakere, oppdatere med oppdaterBrevmottaker og fjerne med fjernBrevmottaker`() {
```

Comment:
```suggestion
    fun `leggTilBrevmottaker skal legge til brevmottaker, oppdatere med oppdaterBrevmottaker og fjerne med fjernBrevmottaker`() {
```

Har type til brevmottaker noe å si for om leggTilBrevmottaker skal legge til eller ikke?
Hvis ikke så synes jeg at man ikke trenger å spesifisere type i test navn